app/templates/suppliers/edit_supplier_organisation_size.html

```diff
@@ -0,0 +1,61 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}Organisation size – Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {% with items = [
+    {
+      "link": "/",
+      "label": "Digital Marketplace"
+    },
+    {
+       "link": url_for('.dashboard'),
+       "label": "Your account"
+    },
+    {
+      "link": url_for('.supplier_details'),
+      "label": "Company details"
+    },
+  ] %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+<div class="single-question-page">
+  {% with heading = "What size is your organisation?",
```

Comment:
Should the heading go inside the `column-two-thirds` div? That's a genuine question - a similar template on the Briefs FE has the div classes nested as `grid-row > column-two-thirds > single-question-page`, but I have no idea what the right order is meant to be.

Reply:
I'm pretty sure the heading should be inside the two-thirds.
Double check with @laurenceberry maybe?

Reply:
This page's HTML was kindly donated by our existing template at https://github.com/alphagov/digitalmarketplace-supplier-frontend/blob/master/app/templates/suppliers/company_name.html, so if it's broken, I'll need to fix it there as well.

Reply:
But going through all the templates in the suppliers folder, `become_a_supplier.html`, `companies_house_number.html`, `company_contact_details.html`, `company_name.html`, (etc I give up now) are all the same.

Reply:
So frigging inconsistent. :sadface:

Reply:
Maybe this single question page is a template that should go out to utils and be generalised in some fashion. 🤷‍♂️

Reply:
The two-thirds column only makes any difference if the heading is long enough to get wrapped by it, so I have a feeling that only long headings ever got moved inside.

Reply:
Fair enough.

Reply:
Although maybe we don't want longer headings to be wrapped at the two-thirds size because it doesn't cause such an accessibility issue with wider headings given their increased font size.

Reply:
I'm pretty sure they make Laurence and Ralph sad though.

Reply:
Happy to put it inside. Can you give some input please @laurenceberry?

Reply:
Sorry for opening a can of worms 😸